.github/workflows/contributed-recipes.yml

```diff
@@ -62,7 +66,30 @@ jobs:
 
       # Not pulling the image, because it might be loaded from previous step or will be downloaded automatically
       - name: Build recipe 🛠
-        run: docker build --rm --force-rm --tag my-custom-image -f ./${{ matrix.dockerfile }} ./
+        if: ${{ matrix.parent-image != '' }}
+        run: |
+          docker build \
+            --rm --force-rm \
+            --tag my-custom-image \
+            -f ./${{ matrix.dockerfile }} \
+            --build-arg BASE_IMAGE=${{ env.REGISTRY }}/${{ env.OWNER }}/${{ matrix.parent-image }} \
```

Comment:
I don't think it's possible to write a ternary here, but if someone knows how do it, it would be great